test/firtool/memoryLowering.fir

```diff
@@ -40,9 +40,9 @@ circuit test:
 ; CHECK-NEXT:     "depth": 16,
 ; CHECK-NEXT:     "width": 32,
 ; CHECK-NEXT:     "masked": true,
-; CHECK-NEXT:     "read": true,
-; CHECK-NEXT:     "write": true,
-; CHECK-NEXT:     "readwrite": false,
+; CHECK-NEXT:     "read": 1,
+; CHECK-NEXT:     "write": 1,
+; CHECK-NEXT:     "readwrite": 0,
```

Comment:
@richardxia: Do you have any idea of the consequences of this change in `seq_mems.json`/`tb_seq_mems.json`?

Reply:
I just took a quick look at our internal codebase, and I think this might actually be safe due to an idiosyncrasy of Python, which is that the integer 0 is considered "falsy" in boolean checks and all other integers are considered "truthy".

That said, if no outside of SiFive is using this file and we consider it to be deprecated, it we may want to err on the conservative side by preserving the original format and emitting a `true` if the number is greater than 0 and `false` otherwise.